spec/spec_helper.rb

```diff
@@ -133,4 +133,24 @@
     example.run
     Timecop.return
   end
+
+  # enable `allow_forgery_protection` in Lighthouse specs to ensure that the endpoints
+  # in those modules have explicitly skipped the CSRF protection functionality
+  lighthouse_dirs = %r{
+    modules/
+    (appeals_api|claims_api|openid_auth|va_facilities|va_forms|vba_documents|
+      veteran|veteran_confirmation|veteran_verification)/
+  }x
+  config.define_derived_metadata(file_path: lighthouse_dirs) do |metadata|
+    metadata[:enable_csrf_protection] = true
+  end
+
+  config.before(:all, :enable_csrf_protection) do
+    @original_allow_forgery_protection = ActionController::Base.allow_forgery_protection
+    ActionController::Base.allow_forgery_protection = true
+  end
+
+  config.after(:all, :enable_csrf_protection) do
+    ActionController::Base.allow_forgery_protection = @original_allow_forgery_protection
+  end
```

Comment:
I think maybe you want `config.before(:suite, ...`, since it will run only once, whereas `:all` will run before each example group.

Reply:
I think you're right.  I was messing with `:all` vs `:suite` earlier, and I thought I changed it.  Obviously I didn't!

Reply:
OK, we were wrong. You can't use `:suite` here:
> WARNING: `:suite` hooks do not support metadata since they apply to the suite as a whole rather than any individual example or example group that has metadata. The metadata you have provided ([:enable_csrf_protection]) will be ignored.